bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -772,9 +772,14 @@ impl Client {
     }
 
     /// Get the preview of a room, to interact with it.
+    ///
+    /// The (optional) list of `via_servers` must be a list of servers that know
```

Comment:
The list isn't optional as in `Option`. Do you mean it can be an empty vector?

Reply:
Yes, the `(optional)` might be redundant because I say it can be empty in the next sentence.

(Thanks for the quick review!)